Automattic/wp-calypso
# Help Center: add forum option

#### Proposed Changes

This adds the Forums as an option to the help center contact page. Previously we only forwarded free users to this option. This adds a new button that shows/hides all the support options with the exception of the Forums at the top.

| Chat Closed | Chat Open |
| - | - |
| <img width="415" alt="Markup 2023-02-09 at 13 13 27" src="https://user-images.githubusercontent.com/33258733/217810255-b8190f65-fa3b-4529-ac8b-b0e324f88c27.png"> | <img width="414" alt="Markup 2023-02-09 at 13 14 33" src="https://user-images.githubusercontent.com/33258733/217810306-cf1240a1-3b8b-4847-b549-cb3c46ca3dab.png"> |

#### Testing Instructions

1. Pull branch and start
2. Open up Help Center and like "Still need help"
3. You should see the forum option regardless of your plan
4. Free users should be forwarded to the forum op
5. Test in atomic using the plugin

[My test topic that was submitted](https://wordpress.com/forums/?post_type=topic&p=3919173&view=all)
